### DISCUSSION TITLE
CI machine is not using latest chrome as it's being used on my local machine

### DISCUSSION THREAD

**Discussion by alex-grinberg** [cypress-io/cypress#27624] [CI setup]
I have a CI machine running Linux Ubuntu that is not using the latest version of Chrome.
When I run the same command on my MacOS, the latest version is being used.
Same code, same repository.
Is there something I can do? I need the latest version in order to run some specific tests that require new features from the latest chrome version

**Comment by jki1** [comment]
Hi @alex-grinberg,

cypress uses the latest available version on your system. If your virtual machine is not updated regularly you need to update the system or manually install browser versions. 

I use a custom docker image for my pipeline runs that takes an image provided by cypress and enhances it by adding the most-recent browser. I have a pipeline schedule to trigger a re-build once a week so that I have a regular update of the browsers in my pipeline.

(You could even add beta versions with the same approach)

```
FROM cypress/included:12.12.0

# Latest chrome
RUN wget --no-verbose -O /usr/src/google-chrome-stable_current_amd64.deb "https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb" && \
  dpkg -i /usr/src/google-chrome-stable_current_amd64.deb ; \
  apt-get install -f -y && \
  rm -f /usr/src/google-chrome-stable_current_amd64.deb`


# Latest firefox
RUN wget --no-verbose -O /tmp/firefox.tar.bz2 "https://download.mozilla.org/?product=firefox-latest&os=linux64&lang=en-US" \
  && tar -C /opt -xjf /tmp/firefox.tar.bz2 \
  && rm /tmp/firefox.tar.bz2 \
  && ln -fs /opt/firefox/firefox /usr/bin/firefox
```

Hope that helps!
